docs/index.md

```diff
@@ -43,7 +43,9 @@ resource "routeros_interface_gre" "gre_hq" {
 
 ### Required
 
-- `hosturl` (String) URL of the ROS router. Include the scheme (http/https)
```

Comment:
That's a bit of a misnomer:
api is the old API without TLS/SSL
apis is old API with TLS/SSL
https is new REST API with TLS/SSL. All three schemes are supported with their own separate ports. So please do not exclude https. 
Correct it in the file https://github.com/terraform-routeros/terraform-provider-routeros/blob/main/templates/index.md.tmpl

Reply:
Thanks. Out of curiosity where does the port get set for https? I was going by: https://github.com/terraform-routeros/terraform-provider-routeros/blob/9124721271cca2e9dc41c027c72f18b3b458f620/routeros/mikrotik_client.go#L84

Reply:
There is no secret, the port is embedded in the schema itself. For REST, a standard http client is used, which parses the passed URL and uses a default port (https = 443).

Reply:
Moved the scheme documentation over to the template and included the https scheme.